docs/attributes-registry/messaging.md

```diff
@@ -63,10 +63,8 @@ size should be used.
 | Value     | Description                                                                                                                                                                                                            | Stability                                                        |
 | --------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------- |
 | `create`  | A message is created. "Create" spans always refer to a single message and are used to provide a unique creation context for messages in batch publishing scenarios.                                                    | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
-| `process` | One or more messages are delivered to or processed by a consumer.                                                                                                                                                      | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
not sure if we strictly need removed operations types but happy to bring them back.